aws/ssm-tomcat-mongodb-replication/mongodb_replication_secondary.json

```diff
@@ -0,0 +1,113 @@
+                        "python get-pip.py\n",
+                        "pip install awscli\n",
+                        "yum -y install firewalld\n",
+                        "sed -i -e 's/SELINUX=enforcing/SELINUX=disabled/g' /etc/sysconfig/selinux\n"
+                    ]
+                }
+            },
+            "nextStep": "changeHostname"
+        },
+        {
+            "name": "changeHostname",
+            "action": "aws:runCommand",
+            "onFailure": "Continue",
+            "inputs": {
+                "DocumentName": "AWS-RunShellScript",
+                "InstanceIds": ["{{instanceIds}}"],
+                "Parameters": {
+                    "commands": [
+                        "sudo su -",
+                        "echo mongodb-secondary > /etc/hostname\n",
+                        "instanceId=$(curl http://169.254.169.254/latest/meta-data/instance-id)\n",
+                        "ipmongoprimary=$(aws ec2 describe-instances --instance-id $instanceId --region ${region} --query 'Reservations[0].Instances[0].Tags[?Key==`ipmongoprimary`].Value' --output text)\n",
+                        "ipmongosecondary=$(aws ec2 describe-instances --instance-id $instanceId --region ${region} --query 'Reservations[0].Instances[0].Tags[?Key==`ipmongosecondary`].Value' --output text)\n",
+                        "echo $ipmongoprimary mongodb-primary >> /etc/hosts\n",
+                        "echo $ipmongosecondary mongodb-secondary >> /etc/hosts\n",
+                        "reboot"
+                    ]
+                }
+            },
+            "nextStep": "waitForInstancesToStartAfterReboot"
+        },
+        {
+            "name": "waitForInstancesToStartAfterReboot",
+            "action": "aws:changeInstanceState",
+            "timeoutSeconds": 600,
+            "onFailure": "Abort",
+            "inputs": {
+                "InstanceIds": [
+                    "{{instanceIds}}"
+                ],
+                "CheckStateOnly": true,
+                "DesiredState": "running"
+            },
+            "nextStep": "installmongoDB"
+        },
+        {
+            "name": "installmongoDB",
+            "action": "aws:runCommand",
+            "onFailure": "Continue",
+            "inputs": {
+                "DocumentName": "AWS-RunShellScript",
+                "InstanceIds": ["{{instanceIds}}"],
+                "Parameters": {
+                    "commands": [
+                        "sudo su -\n",
+                        "systemctl start firewalld\n",
+                        "systemctl enable firewalld\n",
+                        "firewall-cmd --permanent --add-port=22/tcp\n",
+                        "firewall-cmd --permanent --add-port=27017/tcp\n",
+                        "firewall-cmd --reload\n",
+                        "cd ~\n",
+                        "yum install wget vim -y\n",
+                        "wget https://s3.ap-south-1.amazonaws.com/app-server-ilantus/mongo.repo.txt\n",
```

Comment:
"wget https://s3.ap-south-1.amazonaws.com/app-server-ilantus/mongo.repo.txt\n",
ilantus is present.

Reply:
its old one

Reply:
if not required then delete it.